Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM drupal:8.9.1-apache
+FROM drupal:8.9.2-apache
 
 # Install Composer and it's dependencies
 RUN apt-get update && apt-get install -y \
-      curl \
-      git-core \
-      mediainfo \
-      unzip \
-    && rm -rf /var/lib/apt/lists/*
+  curl \
+  git-core \
+  mediainfo \
+  unzip \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
The extra indentation on L5-8 to visually distinguish the `&&`'d command in L9 feels more readable. Happy to see the indentation knocked back one tab, but could we leave that step?

Reply:
Formatting fixed